web/l10n/en.json

```diff
@@ -118,7 +118,7 @@
     "commandDeviceIdentification": "Device Identification",
     "commandIndex": "Index",
     "commandData": "Data",
-    "commandPhoneNumber": "Phone Number",
+    "commandPhone": "Phone",
```

Comment:
I think it might be a good idea to keep text as "Phone Number". It would be clearer.